lib/plugins/aws/invokeLocal/index.js

```diff
@@ -30,7 +30,7 @@ const ensureRuntimeWrappers = isStandalone
         await fse.copy(path.resolve(__dirname, 'runtimeWrappers'), runtimeCachePath);
         return fse.ensureFile(path.resolve(runtimeCachePath, 'validationFile'));
       })
-  : () => Promise.resolve(__dirname);
+  : () => __dirname;
```

Comment:
I believe the function above that is returned if `isStandalone` is true should be marked as async